### PR TITLE
Configure a single pyproject to handle multiple packages

### PR DESCRIPTION
**Pull request by mdantonio**
Merged pyprojects into a single package configured to install all
required subfolders as sub-packages

### CODE REVIEW

**Comment by mdantonio** [comment]
@pdonorio @cassios we are ready to go here

**Review by cassios — APPROVED** [review]
niceee 🏅

File: .github/workflows/mypy.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Install MyPy
         run: |
           pip install mypy
-          pip install ./sdk_commons ./template ./ribbon ./thetanuts
+          pip install . ./template
```
Comment (pdonorio) [inline .github/workflows/mypy.yml:39]: Not in love with the "pip install template".
We can think about having a better way after merging 🤗

**Review by pdonorio — APPROVED** [review]

File: .github/workflows/mypy.yml (hunk 1 above)
Comment (mdantonio) [reply .github/workflows/mypy.yml:39]: ok

**Pull request merged**